src/data/finset/basic.lean

```diff
@@ -836,6 +839,14 @@ theorem union_distrib_right (s t u : finset α) : (s ∩ t) ∪ u = (s ∪ u) 
 
 lemma union_eq_empty_iff (A B : finset α) : A ∪ B = ∅ ↔ A = ∅ ∧ B = ∅ := sup_eq_bot_iff
 
+theorem subset_iff_inter_eq_left (s t : finset α) :
+  s ∩ t = s ↔ s ⊆ t :=
+(inf_eq_left : s ⊓ t = s ↔ s ≤ t)
+
+theorem subset_iff_inter_eq_right (s t : finset α) :
+  t ∩ s = s ↔ s ⊆ t :=
+(inf_eq_right : t ⊓ s = s ↔ s ≤ t)
```

Comment:
These lemmas are in the reverse direction to the set ones. Either you should:
* Change the direction to match
* Change the name to `inter_eq_right`
* Change the direction of the set lemma to match, and change the name of both to `inter_eq_right`, and 

I can't think of a good reason to have different conventions for `finset` and `set`, so I'd prefer option 1 or 3

Reply:
I went with option 3, since it matches the lattice version.  In doing so, I realized that it already used a `_.symm`, so someone must have really wanted the initial ordering.  However, I could not find a usage of the lemmas outside of their file.  CI will tell me if I missed something!